requires/installation.txt

```diff
@@ -1,2 +1 @@
 pika==0.13.1
-tornado>=6,<7
```

Comment:
Considering that sprockets are intended for use with Tornado, it might still be a good idea to require Tornado 6 in installation.txt. It would prevent someone from installing/using this with Tornado 4 or 5, where asyncio support is um not as good.